Extension/src/interfaces.ts

```diff
@@ -0,0 +1,42 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+'use strict';
+
+export interface ICppTools {

```

Comment:
>I [](start = 17, length = 1)

nit: we don't use Hungarian naming conventions in TypeScript.  Please remove the 'I' from the interface name.